src/network/advanceddispatcher.py

```diff
@@ -111,26 +132,32 @@ def handle_read(self):
             self.read_buf.extend(newData)
 
     def handle_write(self):
+        """Send outgoing data from write buffer."""
         self.lastTx = time.time()
         written = self.send(self.write_buf[0:self.uploadChunk])
         asyncore.update_sent(written)
         self.sentBytes += written
         self.slice_write_buf(written)
 
     def handle_connect_event(self):
+        """Callback for connection established event."""
         try:
             asyncore.dispatcher.handle_connect_event(self)
         except socket.error as e:
-            if e.args[0] not in asyncore._DISCONNECTED:
+            if e.args[0] not in asyncore._DISCONNECTED:  # pylint: disable=protected-access
                 raise
 
     def handle_connect(self):
+        """"""
         self.lastTx = time.time()
 
     def state_close(self):
+        """Signal to the processing loop to end."""
+        # pylint: disable=no-self-use
         return False
 
     def handle_close(self):
```

Comment:
Callback for connection being closed, but can also be called directly when you want connection to close.